src/types/course/interfaces/course.interface.ts

```diff
@@ -34,3 +34,9 @@ export interface CourseSection {
   quizzes: string[]
   attachments: string[]
 }
+
+export interface CourseFilters {
```

Comment:
You can try to get these fields using existing Course interface and not create a new interface. I suppose u can use Pick and get these fields with Course interface as generic